node/index.js

```diff
@@ -11,16 +11,35 @@ const speechGoogle = require('./speech_google');
 const speechOpenAI = require('./speech_openai');
 const { state } = require('./state');
 
+// see versions here: https://github.com/wppconnect-team/wa-version/tree/main/html
+const wwebVersion = '2.2412.54';
```

Comment:
Please either write "whatsappWebVersion" or add a comment explaining why "wweb".
Thank you

Reply:
Good point for the name remark, albeit I don't understand this diff: this line and the code that uses it is there since #12